prac_07/convert_miles_km.py

```diff
@@ -0,0 +1,42 @@
+"""Miles to Kilometres Converter"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+MILES_TO_KILOMETERS = 1.60934
+
+
+class MilesConverterApp(App):
+    """Kivy App for converting miles to kilometres."""
+    output_km = StringProperty()
```

Comment:
One of the things we are being marked on is the reusability of our Classes. There is no guarantee that 'output_km' would be suitable if placed into a different program. We should therefore name things so that it describes its purpose, whilst also being transferrable and usable with multiple separate programs (HINT: output_text instead of output_km)